float64[] idar_data
